packages/dashboard/src/components/map-app.tsx

```diff
@@ -41,25 +42,31 @@ const debug = Debug('MapApp');
 const TrajectoryUpdateInterval = 2000;
 // schedule visualizer manages it's own settings so that it doesn't cause a re-render
 // of the whole app when it changes.
-const SettingsKey = 'mapAppSettings';
 const colorManager = new ColorManager();
 
-const DEFAULT_ZOOM_LEVEL = 5;
+const DEFAULT_ZOOM_LEVEL = 20;
 
 function getRobotId(fleetName: string, robotName: string): string {
   return `${fleetName}/${robotName}`;
 }
 
-interface MapSettings {
-  trajectoryTime: number;
-}
-
 export const MapApp = styled(
   createMicroApp('Map', () => {
     const rmf = React.useContext(RmfAppContext);
     const resourceManager = React.useContext(ResourcesContext);
     const [currentLevel, setCurrentLevel] = React.useState<Level | undefined>(undefined);
-    const [disabledLayers, setDisabledLayers] = React.useState<Record<string, boolean>>({});
+    const [disabledLayers, setDisabledLayers] = React.useState<Record<string, boolean>>({
+      Waypoints: false,
+      Dispensers: false,
+      Ingestors: false,
+      Lifts: false,
+      Doors: false,
+      Trajectories: false,
+      Robots: false,
+      Labels: false,
+      PickupLabel: false,
+      DropoffLabel: false,
```

Comment:
```suggestion
      Waypoint labels: false,
      Pickup point labels: false,
      Dropoff point labels: false,
```

My suggestion is not syntactically correct, but  the idea is to have spaces in the layer names.

Reply:
Done 5a39c12c876182611593353d365cb7a307ca6e89 !
I separate the labels in case we have more in the future. For now, I think they are just doors. 

Then the waypoint labels and the dropoff and pickup labels.